postgres/matviews/sms_error_and_users_docs_replication.sql

```diff
@@ -0,0 +1,87 @@
+/* In this view we regroup the metrics of:
+  - sms_error_states
+  - users_replication_limit
+  - replication docs
+*/
+CREATE MATERIALIZED VIEW public.app_monitoring_sms_error_and_users_docs_replication
+TABLESPACE pg_default
+AS
+WITH sms_error_states AS (
```

Comment:
you don't need to do this here - but bonus points if you do

i think the `WITH` and all the `ROW_NUMBER` stuff can be removed entirely with a single `DISTINCT ON` statement at the top

```
  select
    distinct on (partner_name, reported)
    ... selects from sms ... 
    ... your new selects ...
  FROM monitoring_docs AS docs
  INNER JOIN monitoring_urls AS urls ON (docs.url_id=urls.id)
  WHERE doctype = 'monitoring'
```